.claude/skills/logo-grade/scripts/logo_lint.py

```
@@ -0,0 +1,199 @@
+
+
+def local(tag):
+    return tag.rsplit("}", 1)[-1]
+
+
+def lint_file(path):
+    findings = []
+    with open(path, encoding="utf-8", errors="replace") as fh:
+        raw = fh.read()
+    try:
+        root = ET.fromstring(raw)
```

Ruff (0.15.21)

[error] Using `xml` to parse untrusted data is known to be vulnerable to XML attacks; use `defusedxml` equivalents

(S314)